src/app/history/history-bet/history-bet.component.css

```diff
@@ -0,0 +1,42 @@
+:host {
+  min-height: 120px;
+  background-color: var(--background);
+}
+
+.history-title {
+  align-items: center;
```

Comment:
Does this property work since you're not using display: flex (grid)?

Reply:
deleted